### PR TITLE
New Surface Area formula

### PR DESCRIPTION
**Pull request by GigabyteZX1**
I request you to add this PR and give in hacktoberfest label

### CODE REVIEW

**Comment by ChrisO345** [comment]
This isn't your own code.

https://www.geeksforgeeks.org/python-program-to-solve-quadratic-equation/

**Comment by GigabyteZX1** [comment]
> This isn't your own code.
> 
> https://www.geeksforgeeks.org/python-program-to-solve-quadratic-equation/

Extremely Sorry for that i figured the formula used is same so i took the code and made a few adjustments to it. Nonethless i've removed that file and the rest of changes i made are my original changes so i request you to kindly look into that and merge my PR.

File: furthermaths/surface_area.py [hunk 1]
```diff
@@ -102,6 +102,35 @@ def surface_area_cone(radius: float, height: float) -> float:
...
+    :param Height:
+    :return:
+    """
+    if radiusA < 0 or radiusB < 0 or height <0:
+        raise ValueError("surface_area_ovular_cylinder() only accepts non-negative values")
+    lam = (radiusA - radiusB)/(radiusA + radiusB)
+    P = 3.141592653589793*(radiusA + radiusB)*(1 + (3*lam**2)/(10 + (4-3*lam**2)**0.5))
+    A = radiusB*radiusA*3.141592653589793
+    L = P*height
+    return 2*A + L
```
Comment (ChrisO345) [inline furthermaths/surface_area.py:132]: this is the wrong formula

File: furthermaths/surface_area.py [hunk 2]
```diff
@@ -102,6 +102,35 @@ def surface_area_cone(radius: float, height: float) -> float:
...
+    """
+    Finds the surface area of a torus
+
+    :param inner_radius:
+    :param outer_radius:
+    :return:
+    """
+    if inner_radius < 0 or outer_radius < 0:
+        raise ValueError("surface_area_torus() only accepts non-negative values")
+    return 3.141592653589793**2 * (inner_radius + outer_radius) * (outer_radius - inner_radius)
```
Comment (ChrisO345) [inline furthermaths/surface_area.py:115]: Could you use this formula instead as it is more intuitive. Also remember that the outer_radius/major_radius has to be larger than the inner_radius/minor_radius.

**Comment by GigabyteZX1** [comment]
I have made the proposed changes

**Review by ChrisO345 — APPROVED** [review]

**Comment by ChrisO345** [comment]
Thanks so much

**Pull request merged**